content/docs/reactivesearch/v3/list/treelist.md

```diff
@@ -0,0 +1,477 @@
+---
+title: 'TreeList'
+meta_title: 'TreeList'
+meta_description: 'TreeList creates a multiple selection based tree UI component that is connected to a database field.'
+keywords:
+    - reactivesearch
+    - treelist
+    - hierarchicalmenu
+    - appbase
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'web-reactivesearch'
+---
+
+![TreeList image](https://i.imgur.com/AwUfFJ7.png)
+
+`TreeList` creates a selection based hierarchical tree UI component that is connected to a database field. It can support single as well as multiple item selections.
```

Comment:
```suggestion
`TreeList` creates a selection based hierarchical tree UI component that is connected to multiple database fields. It can support single as well as multiple item selections.
```